mono/mini/Makefile.am.in

```diff
@@ -1,6 +1,8 @@
 count=100000
 mtest=for_loop
 monodir=$(top_builddir)
+mono=$(if $(MONO_EXECUTABLE),$(MONO_EXECUTABLE),mono)
+sep=$(if $(LIST_SEPARATOR),$(LIST_SEPARATOR),:)
```

Comment:
And instead of needing to pass this in from the outside, you could do:

````
if HOST_WIN32
PLATFORM_PATH_SEPARATOR=;
else
PLATFORM_PATH_SEPARATOR=:
endif
```